src/services/MovieCritters.Domain/Models/Movie.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace MovieCritters.Domain.Models
+{
+    public class Movie : BaseModel
```

Comment:
Remember to put a string or a file for image in the VO who goes to front

Reply:
Yes, totally forgot that. Done!